### DISCUSSION TITLE
How to create plugin with work with hot reload?

### DISCUSSION THREAD

**Discussion by ko22009** [webpack/webpack#17338] [Q&A]
I create custom plugin which emmit new asset to dist folder.
And in my app I load this file by axios.

Can I apply changes to my app? When I import directly json file, hot reload works, but dynamic in dist folder, doesn't work, but with liveReload is working.

I am using pmmmwh/react-refresh-webpack-plugin and react-refresh/babel for hot reload.

**Comment by alexander-akait** [comment]
You need to write hot code for this. There is a good answer, please read how it works https://stackoverflow.com/questions/24581873/what-exactly-is-hot-module-replacement-in-webpack and how it can be implemented

**Reply by ko22009** [reply]
> You need to write hot code for this. There is a good answer, please read how it works https://stackoverflow.com/questions/24581873/what-exactly-is-hot-module-replacement-in-webpack and how it can be implemented

It's not a answer (

**Reply by alexander-akait** [reply]
What answer do you expect? I don't know your configuration/how you use this plugin, how you import/export and etc, sorry I can't answer better with such information

**Reply by ko22009** [reply]
> What answer do you expect? I don't know your configuration/how you use this plugin, how you import/export and etc, sorry I can't answer better with such information

https://github.com/facebook/react/issues/16604#issuecomment-528663101
